apps/web/src/features/spaces/components/SpacesList/index.tsx

```diff
@@ -74,13 +74,18 @@ const NoSpacesState = () => {
       <Card sx={{ p: 5, textAlign: 'center', width: 1 }}>
         <SpacesIcon />
 
-        <Typography color="text.secondary" mb={1}>
-          No spaces found.
-          <br />
-        </Typography>
-        <Link onClick={() => setIsInfoOpen(true)} href="#">
-          What are spaces?
-        </Link>
+        <Box mb={3}>
+          <Typography color="text.secondary" mb={1}>
+            No spaces found.
+            <br />
+          </Typography>
+          <Link onClick={() => setIsInfoOpen(true)} href="#">
+            What are spaces?
+          </Link>
+        </Box>
+        <Track {...SPACE_EVENTS.CREATE_SPACE_MODAL} label={SPACE_LABELS.space_list_page}>
+          <AddSpaceButton disabled={false} size="medium" />
```

Comment:
Do we only show this if the user is signed in? In that case we can keep it static non disabled